doc/ssl-performance.md

```diff
@@ -36,6 +36,6 @@ Electron | n/a | all | :heavy_check_mark:
 ObjC | Yes | iOS | :x:
 PHP | Yes | all | Same as the `Makefile` case from above
 PHP | No | all | :x:
-Python | n/a | all | :x:
+Python | n/a | Linux, MacOS 64bit | :heavy_check_mark:
```

Comment:
nit: Can we split all options into different entries? It's a bit unclear if Linux 64 bit is supported. Also, the style of this table suggests the platform should be either a specific one or `all`.

Reply:
done.